dt: 0.01
state_publisher_rate: 5.0
led_publisher_rate: 2.0
velocity_publisher_rate: 10.0

crazyflies:
  - name: cf1
    uri: radio://0/80/2M/E7E7E7E7E7
    height: 0.2
    multiranger: True
  - name: cf2
    uri: radio://0/80/2M/E7E7E7E7E8
    height: 0.3
    multiranger: False
  - name: cf3
    uri: radio://0/80/2M/E7E7E7E7E9
    height: 0.2
    multiranger: True
  - name: cf4
    uri: radio://0/80/2M/E7E7E7E7E6
    height: 0.2
    multiranger: False
    